Resources/public/css/theme/views/content-creation.css

```diff
@@ -0,0 +1,199 @@
+/**
+ * Copyright (C) eZ Systems AS. All rights reserved.
+ * For full copyright and license information view LICENSE file distributed with this source code.
+ */
+
+.ez-view-contentcreationview {
+    text-align: left;
+}
+
+.ez-view-contentcreationview .cof-content-creation__header {
+    border-bottom: 1px #000 solid;
+}
+
+.ez-view-contentcreationview .cof-content-creation__header__list {
+    list-style: none;
+}
+
+.ez-view-contentcreationview .cof-btn--change-content-type {
+    background: #fff;
+    border: 1px solid #ccc;
+}
+
+.ez-view-contentcreationview .cof-btn--change-content-type:before,
+.ez-view-contentcreationview .cof-btn--change-content-type:after {
+    content: "";
+    width: 1px;
+    height: 1rem;
+    background: #aaa;
+    position: absolute;
+    top: 50%;
+    right: 1rem;
+}
+
+.ez-view-contentcreationview .cof-btn--change-content-type:before {
+    transform: translate(-50%, -50%) rotate(45deg);
+}
+
+.ez-view-contentcreationview .cof-btn--change-content-type:after {
+    transform: translate(-50%, -50%) rotate(-45deg);
+}
+
+.ez-view-contentcreationview .cof-content-creation__header__text {
+    line-height: 1.5rem;
+    color: #aaa;
+}
+
+.ez-view-contentcreationview .cof-content-creation__header__text:before {
+    position: absolute;
+    top: 0;
+    left: 0;
+    width: 1.5rem;
+    height: 1.5rem;
+    background: #aaa;
+    color: #fff;
+    border-radius: 50%;
+    line-height: 1.5rem;
+    text-align: center;
+}
+
+.ez-view-contentcreationview .cof-content-creation__header__text:first-child:before {
+    content: '1';
+}
+
+.ez-view-contentcreationview .cof-content-creation__header__text:nth-child(2):before {
+    content: '2';
+}
+
+.ez-view-contentcreationview .cof-content-creation__header__text--highlighted {
+    color: #000;
+}
+
+.ez-view-contentcreationview .cof-content-creation__header__text--highlighted:before {
+    background: #3FB499;
+}
+
+.ez-view-contentcreationview .cof-content-creation__page-one,
+.ez-view-contentcreationview .cof-content-creation__page-two {
+    transition: transform .3s cubic-bezier(.25,.8,.25,1);
+}
+
+.ez-view-contentcreationview .cof-content-creation__content-type-selector {
+    background: #fdfdfd;
+}
+
+.ez-view-contentcreationview .cof-content-creation__content-type-selector--is-loading:before {
+    content: "";
+    height: 100%;
+    width: 100%;
+    background: #fdfdfd;
+    position: absolute;
+    top: 0;
+    left: 0;
+    border-radius: 5px;
+}
+
+.ez-view-contentcreationview .cof-content-creation__content-type-selector--is-loading:after {
+    content: '\f110';
+    color: #3FB499;
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    font-family: 'studiofont';
+    animation: spin .7s infinite linear;
+    transform: translate(-50%, -50%);
```

Comment:
As far as I know the `animation` property overrides `transform` property when both applied to the same selector. See: https://github.com/ezsystems/PlatformUIBundle/blob/d38a00b5329c46f6b7d791607b6e3df3eb4858b6/Resources/public/css/theme/app.css#L115